asset/css/markbind.css

```diff
@@ -5,3 +5,14 @@ blockquote {
 .radio-list-item label {
     font-weight: inherit;
 }
+
+@media (max-width: 767px) {
+    .navbar-inverse .navbar-nav .open .dropdown-menu > li > a {
+        color: #000;
```

Comment:
Let's handle `.navbar-default` as well:

```css
.navbar-default .navbar-nav .open .dropdown-menu > li > a,
.navbar-inverse .navbar-nav .open .dropdown-menu > li > a {
    color: #333;
}
```

`#333` is the original color. `unset` also works, but is less explicit.

<img width="310" alt="screenshot 2018-01-19 14 12 06" src="https://user-images.githubusercontent.com/14091939/35137484-9c60974c-fd24-11e7-8257-dfb181f617bb.png">

Reply:
Should we keep the original colour? I feel that it would not be as readable as black on a white background.

Reply:
Original color as in `#333333` right? That's dark enough.

In fact, the "black" that you see for this comment is `#24292e`.

Black as in `#000000` is seldom used due to excessive contrast.
<br />

Related:
- https://ianstormtaylor.com/design-tip-never-use-black/
- https://www.reddit.com/r/web_design/comments/1ovm1o/why_you_should_never_use_black_in_your_designs/

Reply:
Yes that's right!

Good to know! If that's the original colour in the bootstrap CSS then couldn't we just remove these lines?

Reply:
We could. Bootstrap overrides it with `#777777` on mobile, which looks fine.

Reply:
Decided to go with #333333 as #777777 seems a little too light to be used on a white background, and also for consistency with the desktop site.